src/lib/pipe_lib/strict_pipe.mli

```diff
@@ -76,6 +76,11 @@ module Reader : sig
 
     val two : 'a t -> 'a t * 'a t
   end
+
+  val partition_map3 :
```

Comment:
Can you include a doccomment here (you can look at `val fold` for example)
that says that reads have to happen in the same order as writes or else deadlock